pkg/virt-api/api.go

```diff
@@ -265,6 +265,16 @@ func (app *virtAPIApp) composeSubresources() {
 		stopRouteBuilder.ParameterNamed("body").Required(false)
 		subws.Route(stopRouteBuilder)
 
+		subws.Route(subws.GET(definitions.NamespacedResourcePath(subresourcesvmGVR)+definitions.SubResourcePath("expand-flavor")).
+			To(subresourceApp.ExpandFlavorVMRequestHandler).
+			Param(definitions.NamespaceParam(subws)).Param(definitions.NameParam(subws)).
+			Operation(version.Version+"expand-flavor").
```

Comment:
So this name for some reason needs to align with the `SubResourcePath` above otherwise we end up with `503`s returned from somewhere inbetween kube-api and virt-api (or just not logged by the latter). `Operation` also can't conflict any existing routes so this currently causes `generate` to fail given the `PUT` below. I'll leave that for you to address in the morning.